web/js/layers/options.js

```diff
@@ -273,6 +273,7 @@ export function layersOptions(config, models, layer) {
   };
 
   var onRangeUpdate = function () {
+    var current;
```

Comment:
You can add the `var` keyword on line 283 instead, since that's in the middle of the list of vars being declared for this function scope. If you want the variables to be defined first in this function, you can move the `updateRangeLabels()` call below that list.